.travis.yml

```diff
@@ -1,35 +1,41 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "4"
 
 sudo: false
+dist: trusty
+
+addons:
+  chrome: stable
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
```

Comment:
Will the `node_modules` really be cached like this?

Reply:
no idea, this is what `ember init` suggested ;)

Reply:
I don't think it does, since `$HOME/.npm` only contains the globally installed packages. `ember-init` also doesn't include a script..

Reply:
I'd say we leave it as it is for now, since it's the default and we don't expect too many builds in this repo.